docs/kubernetes_deploy.md

```diff
@@ -51,28 +51,9 @@ kube-system   kube-scheduler-user.nvidia.com                   1/1     Running
 kube-system   nvidia-device-plugin-1607379880-dblhc            1/1     Running   0          11m
 ```
 
-Before deploying the model analyzer, you must specify the directories that the model analyzer container will mount and use in `helm-chart/values.yaml`.
+Before deploying the model analyzer, the directories that the container will mount and use must be specified in `helm-chart/values.yaml`.
```

Comment:
"mount and use" is redundant. "that the container will mount" should be sufficient.